CONTRIBUTING.md

```diff
@@ -56,6 +56,35 @@ This project follows the
 [Oracle Java code conventions](http://www.oracle.com/technetwork/java/codeconvtoc-136057.html)
 and the Scala conventions detailed above, preferring the latter.
 
+### Test
+There are two types of tests in this project. Unit tests that are written in Scala, and integration tests that are written in python. 
+We encourage writing integration tests if you have to choose between the two as that helps us ensure the plugin is working across different platforms. 
+
+#### Unit tests
```

Comment:
The bullets in this section are not formatted properly.  See the Markdown rendering at https://github.com/NVIDIA/spark-rapids/blob/a90286e980aa74eff0191d2e496dd8ee6832724b/CONTRIBUTING.md.

In addition the bullet points are quite wordy.  If there's that much to say per bullet, I'd suggest using sub-headings with useful section header names rather than bullets.